docs/Message.mdx

```diff
@@ -0,0 +1,53 @@
+---
+name: Message
+---
+
+import { Playground, PropsTable } from 'docz'
+import { Box, Message, Sans, Spacer } from '../src'
+
+# Message
+
+<Playground>
+
+  <Box width="50%">
+    <Message>
+      Thank you for your order. You’ll receive a confirmation email shortly.
+      If you have questions, please contact <a href="#">orders@artsy.net</a>.
+    </Message>
+
+    <Spacer my={4} />
+
+    <Message size="2">
```

Comment:
should these `size` props be `textSize`?

Reply:
Ah good call! will update.